### PR TITLE
[ISSUE 1036] Reproduce step

### PR DESCRIPTION
**Pull request by CedrickFlocon**
This pr is use to reproduce #1036  issue

> e : apollo-android/apollo-integration/src/test/java/com/apollographql/apollo/Test.kt: (12, 29): Cannot access class 'com.apollographql.apollo.integration.httpcache.fragment.PlanetFragment'. Check your module classpath for missing or conflicting dependencies

### CODE REVIEW

File: apollo-integration/build.gradle [hunk 1]
```diff
@@ -1,52 +1,34 @@
 buildscript {
   dependencies {
-    classpath dep.androidPlugin
     classpath dep.apolloPlugin
   }
 }
 
-apply plugin: 'com.android.application'
+apply plugin: 'java'
+apply plugin: 'kotlin'
```
Comment (sav007) [inline apollo-integration/build.gradle]: not sure but does order of applying plugins matter ?

Comment (CedrickFlocon) [reply apollo-integration/build.gradle]: The order doesn't matter.
I will spend some time next week to have more information to provide

**Comment by digitalbuddha** [comment]
Hi were you able to find any cause?

**Comment by sav007** [comment]
Closing due to inactivity

**Pull request closed without merging**